Tools/InputGeneration/ClusterPointProcess/cluster_point_process.py

```diff
@@ -237,7 +237,7 @@ def add_vertex_events(node, vertex_id, vertex_name, data):
 
     # Get the grid for the Seattle PD Caller Region from the graphml file
     graph_file = '../../gis2graph/graph_files/spd.graphml'
-    spd_cr_id = '194'
+    spd_cr_id = '74'
     graph = nx.read_graphml(graph_file)
     spd_grid = np.array(eval(graph.nodes[spd_cr_id]['segments']))
```

Comment:
This suggests there are a number of magic numbers and hard-coding of things in this code. Do you want to add on as a subsequent issue to make this code more general? That could be a good mini-project for an undergrad.

Reply:
Yes, it has magic numbers, and I will create an issue.